src/cwe_checker_lib/src/utils/binary.rs

```diff
@@ -167,6 +167,22 @@ impl RuntimeMemoryImage {
         Err(anyhow!("Address is not a valid global memory address."))
     }
 
+    /// Read the contents of memory from a given address onwards until a null byte is reached.
```

Comment:
The doc comment is slightly outdated, since the current version additionally checks whether the byte sequence is a valid utf-8 string.